content/general/concepts/software-development-life-cycle/terms/rapid-application-development/rapid-application-development.md

```diff
@@ -0,0 +1,27 @@
+# Rapid Application Development (RAD)
```

Comment:
There needs to be a metadata section. You can see how to build one in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).

You also need to remove the h1 header, it will be provided by the title given in the metadata.

Reply:
> There needs to be a metadata section. You can see how to build one in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata).
> 
> 
> 
> You also need to remove the h1 header, it will be provided by the title given in the metadata.

On it !